manifest.go

```diff
@@ -501,13 +501,11 @@ func fetchManifestEntries(m ManifestFile, fs iceio.IO, discardDeleted bool) ([]M
 	}
 
 	fieldNameToID, fieldIDToLogicalType := getFieldIDMap(sc)
-	isVer1, isFallback := true, false
-	if string(metadata["format-version"]) == "2" {
-		isVer1 = false
-	} else {
+	isFallback := false
+	if string(metadata["format-version"]) != "2" {
```

Comment:
```suggestion
	if string(metadata["format-version"]) == "1" {
```
nit: more explicit, i dont think we'd want this logic for V3